cookbooks/bootstrap/recipes/default.rb

```diff
@@ -0,0 +1,94 @@
+#
+#
+# The bootstrap script for the cookbook-guide
+#
+#
+# What're you looking at? I know what I'm doing.
+#                          - Han Solo
+#
+
+cookbook_file "/tmp/reveal.json" do
+  owner "vagrant"
+  mode "0644"
+  source 'reveal.json'
+end
+
+execute 'Get the markdowns to the correct location' do
```

Comment:
Adding `git` resource to this that pulls down the markdowns then runs the command _maybe_ the correct way to get this done. I'd like to bounce this idea off a couple people first.